bus-mapping/src/operation/container.rs

```diff
@@ -111,6 +114,11 @@ impl OperationContainer {
                 });
                 OperationRef::from((Target::Storage, self.storage.len() - 1))
             }
+            OpEnum::TransientStorage(op) => {
+                // removed rwc_inner_chunk here... does that matter?
+                self.transient_storage.push(Operation::new(rwc, rw, op));
+                OperationRef::from((Target::TransientStorage, self.transient_storage.len() - 1))
+            }
```

Comment:
better to keep rwc check.   (the rwc check is used for early terminate, to avoid rws cost too much memory to make l2geth down.  while for tload/tstore, the 100 gas cost is non trivial, you cannot create too many tload/tstores.  Anyway, keeping rwc_check makes code more consistent.)

Reply:
fixed in latest commit